toolchain/sem_ir/file.h

```diff
@@ -399,13 +344,18 @@ class File : public Printable<File> {
     return value_stores_->strings();
   }
 
-  auto functions_size() const -> int { return functions_.size(); }
-  auto classes_size() const -> int { return classes_.size(); }
+  auto functions() -> ValueStore<FunctionId, Function>& { return functions_; }
+  auto functions() const -> const ValueStore<FunctionId, Function>& {
+    return functions_;
+  }
+  auto classes() -> ValueStore<ClassId, Class>& { return classes_; }
+  auto classes() const -> const ValueStore<ClassId, Class>& { return classes_; }
+  auto types() -> ValueStore<TypeId, TypeInfo>& { return types_; }
+  auto types() const -> const ValueStore<TypeId, TypeInfo>& { return types_; }
```

Comment:
Hm, just noting a change here: previously, `TypeInfo` had been an encapsulated implementation detail here, and the management of the value representation and type completeness wasn't exposed and could only be modified by directly calling `CompleteType`. I think we're more consistent after this change -- we don't have encapsulation for any of the representations of other things like members of `Class` or `Function`, for which we also might want invariants of various kinds.

I think we should have a more general discussion at some point of what we want to do regarding encapsulation of `SemIR` data. Right now, having essentially everything be public seems to be working well, but I do worry that we may have problems down the road when higher-level invariants start becoming more important. But not as a prerequisite for this change :)